uuid/uuid.go

```diff
@@ -27,6 +27,21 @@ func IsAndroid(uuid string) bool {
 	return IsValidFast(uuid) && matchUuidRegexAndroid(uuid) && !isTest(uuid)
 }
 
+type IdentityType int
+
+const (
+	IdentityTypeOther IdentityType = iota
+	IdentityTypeIDFAOrAAID
+	IdentityTypeNoid
+)
+
+func GetIdentity(uuid string) IdentityType {
+	if IsNoid(uuid) {
+		return IdentityTypeNoid
+	}
+	return IdentityTypeIDFAOrAAID
+}
+
 func IsMaid(uuid string) bool {
```

Comment:
This should then also be renamed to `IsIDFAOrAAID`

Reply:
lets remove this at all. I think IsIDFAOrAAID sounds too complicated - we can use `GetIdentity` for that